sos/plugins/openstack_ironic.py

```diff
@@ -63,10 +63,9 @@ def setup(self):
                 "/var/log/containers/httpd/ironic-api/*log"
             ], sizelimit=self.limit)
 
-        self.add_cmd_output('ls -laRt /var/lib/ironic/')
-        self.add_cmd_output(
-            'ls -laRt ' + self.var_puppet_gen + '/var/lib/ironic/'
-        )
+        for path in ['/var/lib/ironic', '/httpboot', '/tftpboot']:
```

Comment:
please also add the expected paths for containerized undercloud. For me, that is

```
# ls /var/lib/config-data/puppet-generated/ironic_inspector/var/lib/ironic/httpboot/
inspector.ipxe
```

Reply:
... or should it be ``/var/lib/config-data/puppet-generated/ironic/var/lib/ironic/httpboot/`` ?